src/pages/organize/[orgId]/projects/[campId]/calendar/tasks/[taskId]/assignees.tsx

```diff
@@ -17,7 +13,9 @@ import TaskAssigneesList from 'features/tasks/components/TaskAssigneesList';
 import useAssignedTasks from 'features/tasks/hooks/useAssignedTasks';
 import { useMessages } from 'core/i18n';
 import { useNumericRouteParams } from 'core/hooks';
+import { useState } from 'react';
```

Comment:
this should be among the external imports 👼

Reply:
In this file there is no distinction between internal and external. This has been the case in a couple of files, and I think it's a separate thing for us to figure out at some point, whether we want to stick to keeping them separate, and if so, enforce it somehow. So I'd rather not make any changes to this right now.